### DISCUSSION TITLE
Feature RCON Console/ Server Commands

### DISCUSSION THREAD

**Discussion by Juli199696** [OpenRCT2/OpenRCT2#15694] [Feature requests]
Hello, i think we definitly need a RCON console for the server admins to change server variables, kick people over console and load new map or restart the server. Also please make it possible to write into the headlessmode of a server console to chat with people and execute commands right into the server that we dont need join the game. Maybe a RCON Tool would be great too.

I think the easiest way to implement something like that is:
1. Create a RCON password and port for the config.
2. Add a "rcon:port passwordhere command" command into the ingame console
3. Add the commands that are needed for server remote access.
 This could be look like 
rcon:8000 123 loadmap newmap.sv6 
After that has been done the server going to load the new map. 

Commands that would be great for that: "restart,loadmap,kick nameofplayer, set serverpassword ,set servername, set rconpassword, forcesave savename. 

Hope that everyone can understand and also this gets into upcomming features :+1:

**Comment by Nubbie** [comment]
Related in some way to #3430
